pyclesperanto_prototype/_tier3/_get_center_of_mass.py

```diff
@@ -0,0 +1,27 @@
+from .._tier0 import Image
+from .._tier0 import create_like
+from .._tier1 import multiply_image_and_coordinate
+from .._tier2 import sum_of_all_pixels
+
+def get_center_of_mass(image : Image):
+    """
+
+    :return:
+    """
+    # TODO: Rename this method to get_centroid
```

Comment:
Or just `centroid`? =) It's better to forget the `get_` prefix in functional settings! It's assumed that you're getting something. =P

Reply:
Yeah. Removing this "get" brings us back to CLIJ1. But I agree. It seems like all methods in clEsperanto will return something